stdlib/pathname/0/pathname.rbs

```diff
@@ -1084,7 +1084,7 @@ class Pathname
   # Note that this method does not handle situations where the case sensitivity of
   # the filesystem in use differs from the operating system default.
   #
-  def relative_path_from: (Pathname | string base_directory) -> Pathname
+  def relative_path_from: (Pathname | (string & _IsA) base_directory) -> Pathname
```

Comment:
Interestingly enough, this is the only function (outside of `JSON` and `Gem::Version`) that needed updating.

Reply:
https://github.com/ruby/ruby/blob/9ee58b2054c1bbe722ae5a2a4ec6a750ee583220/ext/pathname/lib/pathname.rb#L515-L551